duckduckgo/duckduckgo-autofill
# Update icon asset to fix grayscale version

**Reviewer:** @GioSensation 
**Asana:** https://app.asana.com/0/0/1204083883305438/f

## Description
Remove whitespace from base64 image string and update integration test asset

## Steps to test
1. Use this branch to build Firefox extension and load into Firefox
2. Open https://duckduckgo.com/newsletter
3. Confirm grayscale icon showing
